packages/back-end/src/controllers/experiments.ts

```diff
@@ -1442,7 +1449,11 @@ export async function getSnapshotStatus(
         getReportVariations(experiment, phase),
         snapshot.dimension || undefined,
         queryData,
-        snapshot.statsEngine ?? org.settings?.statsEngine
+        snapshot.statsEngine ?? org.settings?.statsEngine,
+        snapshot.sequentialTestingEnabled ??
+          org.settings?.sequentialTestingEnabled,
+        snapshot.sequentialTestingTuningParameter ??
+          org.settings?.sequentialTestingTuningParameter
```

Comment:
Should we switch this function to take an object instead?  I see it's only called 4 places, so won't be a big deal to refactor.